.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,24 +12,19 @@ Make sure you first do a search for existing issues so that is not a duplicate.
 Also go through our FAQ: https://github.com/iTwin/iTwinUI-react/wiki/FAQ
 -->
 
-#### Environment
-- __Package version(s)__: <!-- fill this out -->
-- __Operating System__: <!-- fill this out -->
-- __Browser name and version__: <!-- fill this out -->
+#### Describe the bug (actual behavior)
+<!-- A clear and concise description of what the bug is. -->
 
-#### Code Sandbox
+#### Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+#### Reproduction
 Link to a minimal repro: <!-- fork this sandbox: https://codesandbox.io/s/itwinui-react-example-son74 -->
 
-#### Steps to reproduce
+##### Steps to reproduce
```

Comment:
Why 5?

Reply:
I thought it would make sense as a subheading under "Reproduction"